content/resources/user/gen3-client.md

```diff
@@ -52,60 +53,105 @@ Now you should be able to run the tool on the command-line from any directory by
 
 Windows:
-gen3-client configure --profile bob --cred C:\Users\Bob\Downloads\credentials.json --apiendpoint https://data.mycommons.org
+gen3-client configure --profile=bob --cred=C:\Users\Bob\Downloads\credentials.json --apiendpoint=https://data.mycommons.org
 ```
 
 > __NOTE:__ For these user guides, https://data.mycommons.org is an example URL and will not be the actual URL of the data commons.
 
 When successfully executed, this will create a configuration file, which contains all the API keys and urls associated with each commons profile configured, located in the user folder:
 ```
-Windows: C:\Users\Bob\.gen3\config
 Mac/Linux: /Users/Bob/.gen3/config
+Windows: C:\Users\Bob\.gen3\config
 ```
 > __NOTE:__ These keys must be treated like important passwords; never share the contents of the `credentials.json` or gen3-client `config` file!
 
 * * *
-## 3) Upload a Data File Using a GUID
+## 3) Upload One or More Data File(s) without Using GUID(s)
 * * *
 When metadata records are created in any node in a Gen3 data commons, these records are assigned a unique, 128-bit ID called a ['GUID'](https://dataguids.org/).
 
-GUIDs are generated on the back-end, not submitted by users, and they are stored in the property `object_id`. The GUID or `object_id` for a submitted data file can be obtained via graphQL query or viewing the data file json record in the graphical model of the project.
+GUIDs are generated on the back-end, not submitted by users, and they are stored in the property `object_id`. In the past, GUIDs must be inserted into the common before actual data files can be uploaded. However, a recent update to the Gen3 data commons allows user to submit data files without pre-generated GUIDs. Instead, a random GUID will be generated and assigned to each data file that has been submitted via the `gen3-client upload` command.
+
+Example Usage:
+```
+* For uploading a single file:
+gen3-client upload --profile=<profile_name> --upload-path=<path_to_files/data.bam>
+* For uploading all files within an folder:
+gen3-client upload --profile=<profile_name> --upload-path=<path_to_files/folder/>
+* Can also support regex such as:
+gen3-client upload --profile=<profile_name> --upload-path=<path_to_files/folder/*>
+* Or:
+gen3-client upload --profile=<profile_name> --upload-path=<path_to_files/*/folder/*.bam>
+
+gen3-client upload --profile=bob --file=test.gif
+
+Uploading data ...
+test.gif  3.64 MiB / 3.64 MiB [==========================================================================================] 100.00%
+Successfully uploaded file "test.gif" to GUID 65f5d77c-1b2a-4f41-a2c9-9daed5a59f14.
+1 files uploaded.
+```
+
+### Local submission history:
+In this mode, the application will keeps tracking which local file has already been submitted to avoid potential duplication in submissions. These information are kept in an JSON file under the same user folder as where the `config` file lives, for example:
```

Comment:
"In this mode, the application will keep track of which local files have already been submitted..."

"This information is kept in a JSON file...."